en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-example.md

```diff
@@ -0,0 +1,363 @@
+# Google Pub Sub Connector Example  
+
+The Google Pub/Sub connector allows you to access the [Google Cloud Pub/Sub API Version v1](https://cloud.google.com/pubsub/docs/reference/rest/) through WSO2 EI. Google Cloud Pub/Sub is a fully-managed real-time messaging service that allows you to send and receive messages between independent applications.
+
+## What you'll build
+
+Given below is a sample scenario that demonstrates how to work with the WSO2 EI Google Pub Sub Connector to:
+
+1. Create a Topic for store company update notifications. Once the response from the Google cloud is received, trigger subscription operation using the extracted information from the response.
```

Comment:
for >> to

Reply:
`Once the response from the Google cloud is received, trigger subscription operation using the extracted information from the response` this is done later, isn't it?